operators/src/machine_learning/onnx.rs

```diff
@@ -187,22 +194,10 @@ where
         source_query.attributes = (0..num_bands as u32).collect::<Vec<u32>>().try_into()?;
 
         // TODO: re-use session accross queries?
-        let session = ort::session::Session::builder()
-            .context(Ort)?
-            .commit_from_file(&self.model_path)
-            .context(Ort)
-            .inspect_err(|e| {
-                tracing::debug!(
-                    "Could not create ONNX session for {:?}. Error: {}",
-                    self.model_path.file_name(),
-                    e
-                );
-            })?;
-
-        tracing::debug!(
-            "Created ONNX session for {:?}",
-            &self.model_path.file_name()
-        );
+        let session = load_onnx_model_from_metadata(&self.model_metadata)?;
+
+        // TODO: check @ initialize?
```

Comment:
or: check when adding the model to Geo Engine?

Reply:
[b1facd8](https://github.com/geo-engine/geoengine/pull/1000/commits/b1facd8ff3be816d9088e99465db1d043d4d9230)